test/functional/eval/system_spec.lua

```diff
@@ -446,8 +446,11 @@ describe('systemlist()', function()
     after_each(delete_file(fname))
 
     it('replaces NULs by newline characters', function()
-      if helpers.pending_win32(pending) then return end
-      eq({'part1\npart2\npart3'}, eval('systemlist("cat '..fname..'")'))
+      if iswin() then
+        eq({'part1\npart2\npart3\r'}, eval('systemlist("type '..fname..'")'))
```

Comment:
Should `systemlist` keep the carriage return? If so, should `split(system(), "\n")` be used instead?